packages/shared/src/components/filters/FeedFilters.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import classNames from 'classnames';
+import sizeN from '../../../macros/sizeN.macro';
+import FilterMenu from './FilterMenu';
+import { ClickableText } from '../buttons/ClickableText';
+import FilterIcon from '../../../icons/arrow.svg';
+
+const asideWidth = sizeN(89);
+
+interface FeedFiltersProps {
+  isOpen?: boolean;
+  onBack?: () => void;
+}
+
+let timeout: NodeJS.Timeout;
+
+export default function FeedFilters({
+  isOpen,
+  onBack,
+}: FeedFiltersProps): ReactElement {
+  const [hidden, setHidden] = useState(true);
+
+  useEffect(() => {
+    if (isOpen) {
+      if (timeout) clearTimeout(timeout);
+      setHidden(false);
+      return;
+    }
+
+    timeout = setTimeout(() => setHidden(true), 300);
+  }, [isOpen]);
+
+  return (
+    <aside
+      className={classNames(
+        'fixed top-14 left-0 z-3 bottom-0 self-stretch bg-theme-bg-primary rounded-r-2xl border-t border-r border-theme-divider-primary overflow-y-auto',
+        !isOpen && 'pointer-events-none',
+        hidden && 'invisible',
+      )}
+      style={{
+        width: asideWidth,
+        transform: isOpen ? 'translateX(0)' : `translateX(-${asideWidth})`,
+        willChange: 'transform',
+        transition: 'transform 0.2s linear 0.1s',
```

Comment:
I think we can use these directly from Tailwind.

Reply:
Oh, nice. Will look these up! Thank you!